zh/index.json

```diff
@@ -130,8 +141,12 @@
       "add_to_exclusion_button": "自動添加採礦工具文件夾到排除列表",
       "add_miners_folder_to_exclusion_list_successful": "採礦工具文件夾已成功添加到排除列表中",
       "add_miners_folder_to_exclusion_list_error": "將採礦工具文件夾添加到排除列表時失敗，請再試一次或手動執行",
+      "auto_add_miners_folder_to_exclusion_list_error_title": "Miner exclusion error",
+      "auto_add_miners_folder_to_exclusion_list_error_message": "Error while adding miners folder to Windows Defender's exclussion list, Please click button below to try again or try to restart Crypto Miner",
       "windows_defender": "Windows Defender",
-      "other_virus_software": "其他的防毒軟體"
+      "other_virus_software": "其他的防毒軟體",
+      "restart_crypto_miner_button": "Restart Crypto Miner",
+      "exclusion_confirm_button": "I have added miners folder to my Antivirus software"
```

Comment:
shouldn't we replace crypto miner, windows defender with the translation key directly so we can use the same wording?

Additionally, 
```
"exclusion_confirm_button": "I have added miners folder to my Antivirus software's exclusion list."
```

```
      "auto_add_miners_folder_to_exclusion_list_error_title": "採礦工具錯誤排除",
      "auto_add_miners_folder_to_exclusion_list_error_message": "採礦工具文件夾添加到Windows Defender的排除列表時出錯，請單擊下面的按鈕再試一次或嘗​​試重新啟動Crypto Miner",
      "restart_crypto_miner_button": "重啟Crypto Miner",
      "exclusion_confirm_button": "我已將採礦工具文件夾添加到我的防病毒軟件排除列表"
```